lib/escape.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const escape = require('cross-spawn/lib/util/escape')
+
+/**
+ * Escape a shell argument for Unix shells
+ * @see https://qntm.org/cmd
+ * @param {string} arg the argument
+ * @returns {string} the escaped argument
+ */
+const escapeUnixShellArg = (arg) => `${arg.replace(/([^a-zA-Z0-9_])/g, '\\$1')}`
```

Comment:
Escape anything but alphanumeric characters or the underscore with a `\`, when using Unix shells.

Reply:
Execa also requires double \\ for spaces.